### PR TITLE
REPL/latex_symbols: define commands for « and »

### LINKED ISSUE #50397: REPL tab completions missing double guillemets « and »
Inspired by [a related issue] on the [`julia-vim`][julia-vim] GitHub repository, I am posting this here. In short the REPL tab completions are currently missing the double guillemet symbols `«` and `»`, even though they exist in the [UnicodeData.txt] database as the entries

    00AB;LEFT-POINTING DOUBLE ANGLE QUOTATION MARK;Pi;0;ON;;;;;Y;LEFT POINTING GUILLEMET;;;;

and

    00BB;RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK;Pf;0;ON;;;;;Y;RIGHT POINTING GUILLEMET;;;;

The single guillemets `‹` and `›` are currently available via the commands `\guilsingleft` and `\guilsingright`, in the file `REPL/src/latex_symbols.jl`:

https://github.com/JuliaLang/julia/blob/ecca2c5a2f0719a306c44628736c8f6c8ac6453f/stdlib/REPL/src/latex_symbols.jl#L435-L436

Similar commands `\guildoubleft` and `\guildoubright` might be introduced for double guillemets, as [`julia-vim`][julia-vim] relies on the REPL mappings to produce the corresponding Vimscript mapping.

### Versioninfo
```
Julia Version 1.9.0
Commit 8e630552924 (2023-05-07 11:25 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 4 × Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, skylake)
  Threads: 1 on 4 virtual cores
```
[a related issue]: https://github.com/JuliaEditorSupport/julia-vim/issues/310
[UnicodeData.txt]: https://unicode.org/Public/UNIDATA/UnicodeData.txt
[julia-vim]: https://github.com/JuliaEditorSupport/julia-vim/issues/310

### PR DESCRIPTION
**Pull request by SeSodesa**
- The commands are `\guillemotleft` and `\guillemotright`, respectively.
- Aims to close #50397.

### CODE REVIEW

**Comment by stevengj** [comment]
I guess these are trying to follow the convention set by the existing `\guilsinglleft` and `\guilsinglright` abbreviations for `‹` and `›`, respectively.  Those abbreviations seem to come from the [`csquotes` LaTeX package](https://ctan.org/pkg/csquotes?lang=en).   However `csquotes` defines `\guillemotleft` and `\guillemotright` for `«` and `»`, respectively.   I feel like we should be consistent with the `csquotes` names if we are going to add these.

(Alternatively, the `babel` package uses `\flqq` and `\frqq` for the French/angled double quotes, and `\flq` and `\frq` for the corresponding single quotes.  These are more terse, but I'm not sure that is a big consideration in Julia since none of these are valid identifier characters so they are probably not too common in Julia code.)

**Comment by SeSodesa** [comment]
I'm perfectly cool with `\guillemotleft` and `\guillemotright` for `«` and `»`. I'll make the edit right away, if someone does not beat me to it.

File: stdlib/REPL/src/latex_symbols.jl [hunk 1]
```diff
@@ -434,6 +434,8 @@ const latex_symbols = Dict(
     "\\backprime" => "‵",
     "\\guilsinglleft" => "‹",
```
Comment (stevengj) [inline stdlib/REPL/src/latex_symbols.jl:435]:
```suggestion
    "\\guilsinglleft" => "‹", # note: \guil* quote names follow the LaTeX csquotes package
```

**Review by stevengj — APPROVED** [review]

**Comment by stevengj** [comment]
Marking as "merge me" since this change seems really uncontroversial — we have the single-quote completions, so we should have the corresponding double-quote completions.

**Pull request merged**